reports/performance/_model.qmd

```diff
@@ -137,6 +147,9 @@ generate_model_township_stats <- function(df) {
 
 ## Topline Statistics
 
+IAAO and machine learning standard performance statistics for townships within the ```r metadata$assessment_triad``` triad.
+Statistics highlighted in <span style="background-color:#d3f2c2">green</span> meet the IAAO standard. Note that median absolute percentage error (`MdAPE`) is how most private sector AVMs measure performance.
+
```

Comment:
Added a small snippet of test to the top of each plot/table, particularly in cases where the takeaway from the figure isn't obvious.